modules/azure-public-dns/dns-zones.tf

```
@@ -0,0 +1,11 @@
+ßresource "azurerm_resource_group" "resource_group" {
```

I assume the letter here is a typo?
```suggestion
resource "azurerm_resource_group" "resource_group" {
```

---

Yep, my bad...updating...